.github/workflows/run-terra-featured-workspace-tests.yaml

```diff
@@ -0,0 +1,242 @@
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run Terra Featured Workspace Tests - Batch 3
+        env:
+          SENDGRID_KEY: ${{ steps.secrets.outputs.sendgrid_key }}
+        run: |
+          set -e
+          REPORT_NAME="master_report_batch_2_${CURRENT_DATE}.html"
+          python3 -u featured_workspaces_test.py \
+            -n "$REPORT_NAME" \
+            -b 3 \
+            -v \
+            --abort_hr 2 \
+            --gcs_path "gs://terra-featured-workspace-tests-reports/fw_reports/" \
+            $([[ "${{ github.event.inputs.mute_notifications }}" == 'true' ]] && echo "--mute_notifications") \
+            $([[ "${{ github.event.inputs.troubleshoot }}" == 'true' ]] && echo "--troubleshoot")
+  run-featured-workspace-tests-batch-4:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Generate CURRENT_DATE
+        id: date
+        run: |
+          echo "CURRENT_DATE=$(date +%Y-%m-%d-%H-%M-%S)" >> $GITHUB_ENV
+
+      - name: Authenticate to GCP
+        uses: google-github-actions/auth@v2
+        with:
+          project_id: ${{ env.GCP_PROJECT_ID }}
+          workload_identity_provider: "projects/953233968740/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider"
+          service_account: ${{ env.GCP_SA_EMAIL }}
+
+      - id: secrets
+        uses: google-github-actions/get-secretmanager-secrets@v2
+        with:
+          secrets: |-
+            sendgrid_key:${{ env.GCP_PROJECT_ID }}/sendgrid_key
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run Terra Featured Workspace Tests - Batch 4
+        env:
+          SENDGRID_KEY: ${{ steps.secrets.outputs.sendgrid_key }}
+        run: |
+          set -e
+          REPORT_NAME="master_report_batch_2_${CURRENT_DATE}.html"
+          python3 -u featured_workspaces_test.py \
+            -n "$REPORT_NAME" \
+            -b 4 \
+            -v \
+            --abort_hr 2 \
+            --gcs_path "gs://terra-featured-workspace-tests-reports/fw_reports/" \
+            $([[ "${{ github.event.inputs.mute_notifications }}" == 'true' ]] && echo "--mute_notifications") \
+            $([[ "${{ github.event.inputs.troubleshoot }}" == 'true' ]] && echo "--troubleshoot")
+  notify-slack:
+    runs-on: ubuntu-latest
+    needs: [ run-featured-workspace-tests-batch-1, run-featured-workspace-tests-batch-2, run-featured-workspace-tests-batch-3, run-featured-workspace-tests-batch-4 ]
+    if: ${{ github.event.inputs.mute_notifications != 'true' }}
```

Comment:
in production, IE when you are not testing your changes, should this be "=true" rather than "!=true"?  - basically, don't we want to notify when this is running "for real"?